sphinx_gallery/backreferences.py

```diff
@@ -23,6 +23,19 @@
 from .utils import _replace_md5
 
 
+class DummyClass(object):
+    """Dummy class for testing method resolution."""
+
+    def run(self):
+        """Do nothing."""
+        pass
+
+    @property
+    def prop(self):
+        """Property."""
+        return 'Property'
```

Comment:
This is used for `tinybuild` testing. The cost is a useless entry in our actual official docs but I don't think it's too bad.